cmd/aries-agent-mobile/main.go

```diff
@@ -1,7 +1,7 @@
 /*
 Copyright SecureKey Technologies Inc. All Rights Reserved.
 
-SPDX-License-Identifier: Apache-2.0
+SPDX-License-Identifier: Apache-2.0.
```

Comment:
Probably shouldn't change the license lines ... and, instead, add the license line text to the linter ignore list.

Reply:
probably better to check only `declarations`, what do you think?
It seems like currently, we are using `--all` option https://github.com/tetafro/godot/blob/master/cmd/godot/main.go#L21

Reply:
I have disabled `--all` option. It will check only declarations.